crates/uv/tests/common/mod.rs

```diff
@@ -56,29 +57,75 @@ impl TestContext {
         let cache_dir = assert_fs::TempDir::new().expect("Failed to create cache dir");
         let venv = create_venv(&temp_dir, &cache_dir, python_version);
 
+        // The workspace root directory is not available without walking up the tree
+        // https://github.com/rust-lang/cargo/issues/3946
+        let workspace_root = Path::new(&std::env::var("CARGO_MANIFEST_DIR").unwrap())
+            .join("..")
+            .join("..");
+
+        let site_packages = site_packages_path(&venv, format!("python{python_version}"));
+
         let mut filters = Vec::new();
         filters.extend(
             Self::path_patterns(&cache_dir)
                 .into_iter()
                 .map(|pattern| (pattern, "[CACHE_DIR]/".to_string())),
         );
         filters.extend(
-            Self::path_patterns(&temp_dir)
+            Self::path_patterns(&site_packages)
                 .into_iter()
-                .map(|pattern| (pattern, "[TEMP_DIR]/".to_string())),
+                .map(|pattern| (pattern, "[SITE_PACKAGES]/".to_string())),
         );
```

Comment:
Added a `SITE_PACKAGES` filter which is helpful for cross-platform tests with package install paths in them.